modules/admin_manual/pages/configuration/server/occ_commands/app_commands/files_lifecycle.adoc

```diff
@@ -0,0 +1,202 @@
+
+The `lifecycle` commands configure the File Lifecycle Management app.
+
+.App Configuration
+[source,console]
+----
+lifecycle
+ lifecycle:archive          Archive files which have reached a certain age
+ lifecycle:expire           Expire files from Archive which have reached a certain age
+ lifecycle:restore          Restore files from Archive to the original location
+ lifecycle:restore-all      Restore all archived files in the system back to their
+                            original locations
+ lifecycle:set-upload-time  Set upload time for files which do not have one
+----
+
+`config:app` commands configure the Policies for the File Lifecycle Management app.
+
+
+.Policy Configuration
+[source,console]
+----
+ config:app:get|set
+ files_lifecycle archive_period   Number of days passed after upload (or restore)
+                                  that files will be archived
+ files_lifecycle expire_period    Number of days passed after archiving that files will
+                                  be permanently deleted
+ files_lifecycle excluded_groups  Define groups of users that are exempt from the
+                                  Lifecycle policies
+ files_lifecycle policy           Restoration Policies for Users
+ files_lifecycle disable_ui       Enable/Disable the user interface components
+----
+
+== App Configuration
+
+== Archive Aged Files
+
+Archive files which have reached a certain age.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} lifecycle:archive [options]
+----
+
+=== Options
+
+[width="90%",cols="40%,80%",]
+|===
+| `-d, +
+--dryrun[=DRYRUN]` 
+| Dont apply changes to the system [default: false]
+|===
+
+== Expire Files From Archive
+
+Expire files from archive which have reached a certain age.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} lifecycle:expire [options]
+----
+
+=== Options
+
+[width="90%",cols="40%,80%",]
+|===
+| `-d, +
+--dryrun[=DRYRUN]` 
+| Dont apply changes to the system [default: false]
+|===
+
+== Restore Files From Archive
+
+Restore files from archive to the original location.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} lifecycle:restore <path>
+----
+
+=== Arguments
+
+[width="90%",cols="40%,80%",]
+|===
+| `path` 
+| Enter path to a folder or to a single file
+|===
+
+===Example
+
+Restore all files in folder `project1` for user alice `/work/projects/project1`
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} lifecycle:restore /alice/archive/files/work/projects/project1
+----
+
+== Restore All Files From Archive
+
+Restore all archived files for all users in the system back to their original locations. This command has no additional arguments or options.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} lifecycle:restore-all
+----
+
+== Set Default Upload Time
+
+Set upload time for files which do not have one.
```

Comment:
Files without upload time are silently skipped by `occ lifecycle:archive`
E.g. files that were uploaded before the files_lifecycle app was configured (or when it is temporarily disabled) do not have an upload time.

Reply:
fixed